packages/text-select/src/token-select/index.jsx

```diff
@@ -16,9 +17,9 @@ export class TokenSelect extends React.Component {
     classes: PropTypes.object.isRequired,
     onChange: PropTypes.func.isRequired,
     disabled: PropTypes.bool,
-    tokenComponent: PropTypes.oneOfType([PropTypes.element, PropTypes.func]),
     highlightChoices: PropTypes.bool,
-    maxNoOfSelections: PropTypes.number
+    maxNoOfSelections: PropTypes.number,
+    showCorrectnessToken: PropTypes.bool
```

Comment:
remove `showCorrectnessToken` - instead check that correct is either `true` or `false`?